life_expectancy/cleaning.py

```diff
@@ -11,9 +12,12 @@
 SAVE_FILE_NAME = "pt_life_expectancy.csv"
 
 
-def clean_data(region_name: str):  # pylint: disable=C0116
-
+def load_data() -> pd.DataFrame:  # pylint: disable=C0116
```

Comment:
Nice use of typehints, I forgot to use it in a couple of my funcs, I'll adapt my code based on yours (: